src/main/java/laddergame/controller/LadderGameController.java

```diff
@@ -0,0 +1,37 @@
+package laddergame.controller;
+
+import laddergame.model.Height;
+import laddergame.model.Ladder;
+import laddergame.model.Persons;
+import laddergame.view.InputView;
+import laddergame.view.OutputView;
+
+public class LadderGameController {
+    InputView inputView = new InputView();
+    OutputView outputView = new OutputView();
```

Comment:
```suggestion
public class LadderGameController {
    private final InputView inputView = new InputView();
    private final OutputView outputView = new OutputView();
```

외부에서 사용하는 필드가 아니므로 숨겨주면 더 좋겠죠?

Reply:
코드 상에 접근 제한자가 빠진 부분이나, 순서가 뒤죽박죽인 부분이 많다는 걸 확인하게 되었습니다 🥲
처음 PR을 보낼 때는 왜 보이지 않았던 건지,, 앞으로는 더 꼼꼼하게 확인하는 습관을 들여야겠습니다 🥺